### PR TITLE
Update K8s quickstart to reflect Dashboard update

### PR DESCRIPTION
**Pull request by dalees**

### CODE REVIEW

File: source/kubernetes/quickstart.rst [hunk 1]
```diff
@@ -193,91 +173,60 @@ to interact with your Catalyst Cloud Kubernetes cluster using the command line.
...
-
-      Now define the ``KUBECONFIG`` environment variable, to configure ``kubectl``
-      to connect to your cluster.
+      Define the ``KUBECONFIG`` environment variable, to configure ``kubectl``
+      to connect to your cluster using the downloaded file.
 
       .. code-block:: bat
 
-        set KUBECONFIG=%cd%\config
+        set KUBECONFIG=\Users\<username>\Downloads\quickstart1_kubeconfig
```
Comment (Callum027) [inline source/kubernetes/quickstart.rst:222]:
```suggestion
        set KUBECONFIG=%homedrive%%homepath%\Downloads\quickstart1_kubeconfig
```

File: source/kubernetes/quickstart.rst [hunk 2]
```diff
@@ -193,91 +173,60 @@ to interact with your Catalyst Cloud Kubernetes cluster using the command line.
...
+    .. group-tab:: Windows (PowerShell)
 
-      Now define the ``KUBECONFIG`` environment variable, to configure ``kubectl``
+      In a PowerShell environment, define the ``KUBECONFIG`` environment variable, to configure ``kubectl``
       to connect to your cluster.
 
       .. code-block:: powershell
 
-        $Env:KUBECONFIG = $pwd\config
+        $Env:KUBECONFIG = \Users\<username>\Downloads\quickstart1_kubeconfig
```
Comment (Callum027) [inline source/kubernetes/quickstart.rst:213]:
```suggestion
        $Env:KUBECONFIG = $env:USERPROFILE\Downloads\quickstart1_kubeconfig
```

File: source/kubernetes/quickstart.rst [hunk 3]
```diff
@@ -193,91 +173,60 @@ to interact with your Catalyst Cloud Kubernetes cluster using the command line.
...
+      environment variable, to configure ``kubectl`` to connect to your cluster.
 
       .. code-block:: bash
 
-        openstack coe cluster config quickstart-cluster --use-keystone --dir "$(pwd)"
+        export KUBECONFIG="${HOME}/Downloads/quickstart1_kubeconfig"
 
-      Now export the ``KUBECONFIG`` environment variable, to configure ``kubectl``
-      to connect to your cluster.
+      Alternatively, if you have no other Kubernetes clusters, you could move the downloaded file `quickstart1_kubeconfig` to the default location for ``kubectl``, in `$HOME/.kube/config`.
```
Comment (Callum027) [inline source/kubernetes/quickstart.rst:199]:
```suggestion
      Alternatively, if you have no other Kubernetes clusters, you could move the downloaded file
      (``quickstart1_kubeconfig``) to the default location for ``kubectl``, in ``$HOME/.kube/config``.
```

File: source/kubernetes/quickstart.rst [hunk 4]
```diff
@@ -193,91 +173,60 @@ to interact with your Catalyst Cloud Kubernetes cluster using the command line.
...
 
-        export KUBECONFIG="$(pwd)/config"
-
-    .. group-tab:: Windows (PowerShell)
-
-      .. note::
-
-        In the below examples, the kubeconfig file will be created in the current folder
-        of your terminal session.
+        mv ${HOME}/Downloads/quickstart1_kubeconfig ${HOME}/.kube/config
```
Comment (Callum027) [inline source/kubernetes/quickstart.rst:203]: Ensure `${HOME}/.kube` exists. Also, quote things to protect against spaces causing problems.

```suggestion
        mkdir -p "${HOME}/.kube/config" && mv "${HOME}/Downloads/quickstart1_kubeconfig" "${HOME}/.kube/config"
```

File: source/kubernetes/quickstart.rst [hunk 5]
```diff
@@ -193,91 +173,60 @@ to interact with your Catalyst Cloud Kubernetes cluster using the command line.
...
 
-First, run the following commands to create the **kubeconfig** file.
+First, ensure you have downloaded the **kubeconfig** file from the Catalyst Cloud
+dashboard using the **Download Kubeconfig** button.
+
+.. note::
+
+  Your browser may download the **kubeconfig** file to another location. Specific
+  browser download locations are out of scope for this document, you can also locate these
+  within your browser.
```
Comment (Callum027) [inline source/kubernetes/quickstart.rst:183]:
```suggestion
  Your browser may download the **kubeconfig** file to another location.
  If you wish to download the **kubeconfig** file to a specific location
  (and not the default location for your operating system), replace ``.../Downloads``
  in the below file paths with your desired download location.
```

File: source/kubernetes/quickstart.rst [hunk 6]
```diff
@@ -193,91 +173,60 @@ to interact with your Catalyst Cloud Kubernetes cluster using the command line.
...
-      The following command will create the kubeconfig in the target directory
-      with the filename ``config``.
-
-      .. code-block:: powershell
-
-        openstack coe cluster config quickstart-cluster --use-keystone --dir $pwd
+    .. group-tab:: Windows (PowerShell)
 
-      Now define the ``KUBECONFIG`` environment variable, to configure ``kubectl``
+      In a PowerShell environment, define the ``KUBECONFIG`` environment variable, to configure ``kubectl``
```
Comment (Callum027) [inline source/kubernetes/quickstart.rst:208]:
```suggestion
      In a PowerShell environment, define the ``KUBECONFIG`` environment variable to configure ``kubectl``
```

File: source/kubernetes/quickstart.rst [hunk 7]
```diff
@@ -132,34 +126,19 @@ can also be configured individually. For this guide we will use the default valu
...
-    :align: center
-
-That should be everything you need to configure,
+That is everything you need to configure for this guide,
 so press the **Submit** button to create the cluster.
 
 You will be returned to the **Clusters** page, where you can monitor the state
 of your Kubernetes clusters. Our new cluster should be listed,
-in ``CREATE_IN_PROGRESS`` state.
+in ``CREATE_IN_PROGRESS`` state, refresh the browser to see the updated status.
```
Comment (Callum027) [inline source/kubernetes/quickstart.rst:141]:
```suggestion
in ``CREATE_IN_PROGRESS`` state. If the cluster is not yet listed, refresh the page.
```

File: source/kubernetes/quickstart.rst [hunk 8]
```diff
@@ -132,34 +126,19 @@ can also be configured individually. For this guide we will use the default valu
 
 Select the **Network** tab to configure the cluster's network access.
 
-A few options are available here, but for this guide, just make sure that
-**Create New Network** is checked (the default), to create a new virtual network for the cluster.
+To enable access to the Kubernetes API from the public Internet, select
+'Accessible with public floating IP' from the **Floating IP** drop-down list.
+For the quickstart, we will leave the **Allowed CIDRs** field empty.
```
Comment (Callum027) [inline source/kubernetes/quickstart.rst:131]:
```suggestion
For this guide, we will leave the **Allowed CIDRs** field empty.
```

**Review by Callum027 — CHANGES_REQUESTED** [review]
Some minor nitpicks, overall looks like it reflects the changes to the UI well.

`_containers_assets/quickstart-advanced.png` needs to be removed, as well.

File: source/kubernetes/quickstart.rst (hunk 4 above)
Comment (Callum027) [reply source/kubernetes/quickstart.rst:203]: This suggestion was slightly incorrect. Just updated it.

File: source/kubernetes/quickstart.rst (hunk 7 above)
Comment (dalees) [reply source/kubernetes/quickstart.rst:141]: Even if it is listed, the status won't update to `CREATE_COMPLETE` without refreshing. I'll have a think about the wording on this.

Comment (Callum027) [reply source/kubernetes/quickstart.rst:141]: You're right. Guess we should also consider putting in an auto-refresh for this on the dashboard side, as well (if possible).

Comment (dalees) [reply source/kubernetes/quickstart.rst]: Updated

**Review by Callum027 — APPROVED** [review]

**Pull request merged**